action.yml

```diff
@@ -48,23 +48,23 @@ inputs:
     type: string
     description: 'The name of the moleculer services you want to deploy, it  should between {}'
     required: false
-    default: '{API, test}'
+    default: ''
   SERVICEDIR:
     type: string
     description: 'The name of the moleculer services directory'
     required: false
     default: 'build/services'
   MONGO_URI:
     description: 'The DB connection string'
-    required: false
+    required: true
     default: ''
   HELM_SET:
     description: Additional helm values to set environment variables (corresponds to `helm upgrade --set`). Should have format environment.env.VAR1=VAL1,environment.env.VAR2=VAL2.
     required: false
   BASE_URL:
     description: "Your base URL for your deployment"
-    default: 'x.knawat.com'
-    required: false
+    default: 'x.example.com'
```

Comment:
if this should be a default value to avoid broken runs why we do not replace it with a real thing that can really be used as the default? such as local docker link?